src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -286,13 +286,14 @@ Referenced assemblies:
 - 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
 - 'PresentationCore, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
 - 'PresentationFramework, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
+- 'SonarLint.VisualStudio.ConnectedMode, Version=6.17.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
```

Comment:
I'll add a specific project reference from `TeamExplorer` to `ConnectedMode` in a separate commit.